src/MeetingsSDKAdapter.js

```diff
@@ -180,30 +189,59 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
   }
 
   /**
-   * Returns a promise to a local media stream based on the given constraints.
+   * Returns a promise to a local media stream based on the given mediaDirection, audioVideo options.
    *
    * @see {@link MediaStream|https://developer.mozilla.org/en-US/docs/Web/API/MediaStream}.
    *
    * @private
    * @param {string} ID ID of the meeting for which to fetch streams
-   * @param {object} constraint Media stream constraints
+   * @param {object} mediaDirection A configurable options object for joining a meetings
+   * @param {object} audioVideo audio/video object to set audioinput and videoinput devices
    * @returns {Promise.<MediaStream>} Promise to requested media stream
    */
-  async getStream(ID, constraint) {
+  async getStream(ID, mediaDirection, audioVideo) {
     let localStream = null;
 
     try {
       const sdkMeeting = this.fetchMeeting(ID);
 
-      [localStream] = await sdkMeeting.getMediaStreams(constraint);
+      [localStream] = await sdkMeeting.getMediaStreams(mediaDirection, audioVideo);
     } catch (error) {
       // eslint-disable-next-line no-console
-      console.error(`Unable to retrieve local media stream for meeting "${ID}" with constraint "${constraint}"`, error);
+      console.error(`Unable to retrieve local media stream for meeting "${ID}" with mediaDirection "${mediaDirection}" and audioVideo "${audioVideo}"`, error);
```

Comment:
are `mediaDirection` and `audioVideo` displayed in a readable way in the console? or just [object Object]? for readability, you can add them as separate args:
```js
'Unable to retrieve...', ID, 'with mediaDirection', mediaDirection, 'and audioVideo', audioVideo